bin/install-theos

```diff
@@ -220,13 +220,26 @@ darwin_mobile() {
 		exit 3
 	fi
 
+	# Compatbility check
+	APTVER="$(apt-get --version | head -n1 | cut -d' ' -f2)"
+	if [[ $(dpkg --compare-versions $APTVER ge 1.1; echo $?) == 0 ]]; then
```

Comment:
nit: I think this can be simplified to `if dpkg --compare-versions $APTVER ge 1.1; then`

Reply:
Good call, thanks.